### DISCUSSION TITLE
INFO  Validating config FATAL,no more error.

### DISCUSSION THREAD

**Discussion by FreeDivers** [hexojs/hexo#5038] [Question / Answer] [answered]
I saw this https://github.com/hexojs/hexo/discussions/5032, but I have been the latest node.js
now i can't use hexo commands.
[![b9GBd.png](https://s1.328888.xyz/2022/08/22/b9GBd.png)](https://imgloc.com/i/b9GBd)

**Comment by yoshinorin** [comment, marked as answer]
refs: https://github.com/hexojs/hexo/issues/5039
